tools/configure_server_zone.rb

```diff
@@ -0,0 +1,29 @@
+if ARGV.empty?
+  puts "USAGE: #{__FILE__} server_id zone_name"
```

Comment:
This usage doesn't include `bin/rails r` to invoke it.  Note, we should require config/environment, so this will be correct once that's done (config/environment, shebang, chmod +x).